sparsity/test/test_dask_sparse_frame.py

```diff
@@ -0,0 +1,94 @@
+import shutil
+import tempfile
+import os
+from contextlib import contextmanager
+
+import dask
+import pytest
+
+import sparsity as sp
+import sparsity.dask as dsp
+import pandas as pd
+import numpy as np
+import dask.dataframe as dd
+
+from sparsity.dask.reshape import one_hot_encode
+
+dask.context.set_options(get=dask.async.get_sync)
+
+
+@contextmanager
+def tmpdir(dir=None):
+    dirname = tempfile.mkdtemp(dir=dir)
+
+    try:
+        yield dirname
+    finally:
+        if os.path.exists(dirname):
+            shutil.rmtree(dirname, ignore_errors=True)
+
+
+
+def test_from_pandas():
+    dsf = dsp.from_pandas(pd.DataFrame(np.random.rand(10,2)),
+                          npartitions=3)
+    res = dsf.compute()
+
+    assert isinstance(res, sp.SparseFrame)
+    assert res.shape == (10,2)
+
+
+def test_map_partitions():
+    data = pd.DataFrame(np.random.rand(10, 2))
+    dsf = dsp.from_pandas(data,
+                          npartitions=3)
+    dsf = dsf.map_partitions(lambda x: x, dsf._meta)
```

Comment:
It could but for now we need to pass it. Except someone can take the time to implement it